lib/tests/test_revset.rs

```diff
@@ -3049,6 +3052,54 @@ fn test_evaluate_expression_mine() {
     );
 }
 
+#[test]
+fn test_evaluate_expression_signed() {
+    let signer = Signer::new(Some(Box::new(TestSigningBackend)), vec![]);
+    let settings = testutils::user_settings();
+    let test_workspace =
+        TestWorkspace::init_with_backend_and_signer(TestRepoBackend::Git, signer, &settings);
```

Comment:
nit: if we don't have a reason to use the `Git` backend, `TestRepoBackend::Test` is preferred.